### ISSUE TITLE
setting pretraining learning rate in command line interface

### ISSUE THREAD

**Issue by 2533245542** [Lightning-AI/litgpt#1905]
I tried to set it by 

`litgpt pretrain --config pythia14m.yaml --optimizer.class_path torch.optim.AdamW --optimizer.init_args.lr 1e-5`

but not working

**Comment by Kiwy3** [comment]
Hello,

To define optimizer parameters, the `--optimizer` argument takes dictionnary. I had the same problem, you can use for instance
`
-- optimizer '{'class_path': 'torch.optim.AdamW', 'init_args': {'lr': 0.001, 'weight_decay': 1e-05, 'betas': [0.9, 0.999]}} '
`
